skyportal_fink_client/skyportal_fink_client.py

```diff
@@ -0,0 +1,150 @@
+# coding: utf-8
+import os
+import yaml
+from fink_client.consumer import AlertConsumer
+from astropy.time import Time
+
+import utils.skyportal_api as skyportal_api
+
+# open yaml config file
+with open(
+    os.path.abspath(os.path.join(os.path.dirname(__file__))) + "/../config.yaml", "r"
+) as stream:
+    try:
+        conf = yaml.safe_load(stream)
+    except yaml.YAMLError as exc:
+        print(exc)
+
+
+def fid_to_filter(fid: int):
```

Comment:
Just to prepare for the future -- The conversion dic is harcoded here, and specific to ZTF. In a not-so-long time, Fink will listen to LSST alerts (and others). Two options here: (1) either you make the function name explicit `fid_to_filter_ztf`, or you expand the constructor `def fid_to_filter(fid: int, switcher: dic = {1: "ztfg", 2: "ztfr", 3: "ztfi"}):` and the logic inside the function.